### ISSUE TITLE
ENABLE_OSS is commented out in CMakeLists.txt

### ISSUE THREAD

**Issue by yurivict** [thestk/stk#141]
GNU configure has --enable-oss present.

**Comment by garyscavone** [comment]
Yes, theoretically OSS support could be compiled, though I doubt it will work because I haven't been able to test it for a decade or more. Are you suggesting that option be removed?

**Comment by yurivict** [comment]
GNU configure supports the oss option and the cmake script doesn't.

FreeBSD, for example, does support OSS. which is a default audio backend there. Jack, PulseAudio, etc. are also supported, on top of OSS.

If OSS can be supported easily - then it should be enabled in both GNU configure and cmake scripts.
But if OSS is irreparably broken - then it should be removed from both.
